airflow/providers/google/cloud/sensors/bigquery.py

```diff
@@ -256,6 +256,11 @@ class BigQueryTableExistenceAsyncSensor(BigQueryTableExistenceSensor):
     """
     Checks for the existence of a table in Google Big Query.
 
+    This class is deprecated and will be removed in a future release.
+
+    Please use :class:`airflow.providers.google.cloud.sensors.bigquery.BigQueryTableExistenceSensor`
+    and set `deferrable` attribute to `True` instead.
```

Comment:
```suggestion
    and set *deferrable* to *True* instead.
```

Same for other occurances. Single backtick in restructuredText is a footgun.